src/components/v2/values/chartValuesDiff/ChartValuesEditor.tsx

```diff
@@ -253,9 +253,12 @@ export default function ChartValuesEditor({
                         .then((res) => {
                             const _valuesForDiff = valuesForDiffState.valuesForDiff
                             const _manifestsForDiff = valuesForDiffState.manifestsForDiff
-                            const _selectedValues = isExternalApp
-                                ? YAML.stringify(JSON.parse(res.result.valuesYaml))
-                                : res.result.valuesYaml
+                            let _selectedValues : string
```

Comment:
Removing `isExternalApp` check was intentional?

Reply:
yeah. It was the reason of the reported bug